.github/style.css

```diff
@@ -0,0 +1,111 @@
+body{
+background-image: url("https://www.transparenttextures.com/patterns/cream-paper.png");
+background-color:#007a99;
+}
+
+.container{
+    width: 250px;
+    position: fixed;
```

Comment:
Giving this container a fixed width and having wrap on the contents is what's making the buttons wrap to the second line.